db/migrations/20230104042228_create_plantsimg_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE plantsimg (
```

Comment:
```suggestion
CREATE TABLE plant_images (
```
하나의 식물에 대한 이미지들이므로 다음과 같이 수정되어야겠네요!